test/Microsoft.ML.OnnxTransformerTest/OnnxTransformTests.cs

```diff
@@ -535,8 +541,8 @@ public void OnnxModelInMemoryImage()
             // Create in-memory data points. Its Image/Scores field is the input/output of the used ONNX model.
             var dataPoints = new ImageDataPoint[]
             {
-                new ImageDataPoint(Color.Red),
-                new ImageDataPoint(Color.Green)
+                new ImageDataPoint(red: 255, green: 0, blue: 0),
+                new ImageDataPoint(red: 0, green: 255, blue: 0),
```

Comment:
This green is a different green than I commented on above.

Reply:
Good point, green should be `128` but really this value doesn't matter as `255` value still work. I'll change it anyway.